### PR TITLE
consolidations, withdrawals: output request type

### PR DESCRIPTION
**Pull request by fjl**
This changes the contracts to add the request type into the output buffer.

### CODE REVIEW

**Comment by fjl** [comment]
Note the type is not added into the log data.

**Review by mkalinin — APPROVED** [review]
LGTM!

**Comment by fjl** [comment]
Needs some more updates to the tests. It's going to need a lot of changes to the consolidations tests.

File: src/consolidations/main.eas [hunk 1]
```diff
@@ -26,16 +26,18 @@
 #define FEE_UPDATE_FRACTION 17
 #define EXCESS_INHIBITOR 1181
 
-#define INPUT_SIZE 96    ;; the size of (source ++ target)
-#define RECORD_SIZE 116  ;; the size of (address ++ source ++ target)
+#define INPUT_SIZE 96          ;; source ++ target
+#define LOG_RECORD_SIZE 116    ;; address ++ source ++ target)
```
Comment (chfast) [inline src/consolidations/main.eas]:
```suggestion
#define LOG_RECORD_SIZE 116    ;; address ++ source ++ target
```

File: src/consolidations/main.eas [hunk 2]
```diff
@@ -216,17 +217,17 @@ begin_loop:
   push0                 ;; [i, count, head_idx, tail_idx]
 
 accum_loop:
-  ;; This loop will read each request and byte bang it into a RECORD_SIZE byte chunk.
+  ;; This loop will read each request and byte bang it into a OUT_RECORD_SIZE byte chunk.
```
Comment (lightclient) [inline src/consolidations/main.eas]:
```suggestion
  ;; This loop will read each request and byte bang it into an OUT_RECORD_SIZE byte chunk.
```

**Review by lightclient — APPROVED** [review]

**Comment by fjl** [comment]
Closing in favor of #25

**Pull request closed without merging**